### PR TITLE
Add codeforces extension

### PR DESCRIPTION
**Pull request by Iron0utlaw**
## Description

Extension to have a quick glance over a [Codeforces](https://codeforces.com/) Handle providing details such as their Rating, Participated Contest and Submissions in a Contest. It uses[ Codeforces API](https://codeforces.com/apiHelp) to fetch the details

## Screencast

Screenshot 2024-01-01 at 22 39 18
Screenshot 2024-01-01 at 22 39 35
Screenshot 2024-01-01 at 22 39 46
Screenshot 2024-01-02 at 21 04 44

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Iron0utlaw** [comment]
I have fixed lint issue on my machine but when I try to publish it again I receive a error
```Found & ignored ./node_modules ; is listed in .gitignore
Found & ignored ./raycast-env.d.ts ; is listed in .gitignore

Starting: publish
node:internal/modules/cjs/loader:1042
  throw err;
  ^

Error: Cannot find module '/Users/harsh/Code/Raycast Extensions/codeforces-extension/publish'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)
    at Module._load (node:internal/modules/cjs/loader:885:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.13.0

File: extensions/codeforces-extension/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "codeforces-extension",
+  "title": "Codeforces-Extension",
```
Comment (pernielsentikaer) [inline extensions/codeforces-extension/package.json:4]:
```suggestion
  "title": "Codeforces",
```

File: extensions/codeforces-extension/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  "author": "harsh_varshney",
+  "categories": [
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-handle",
+      "title": "Search Handle",
+      "subtitle": "Search a handle",
```
Comment (pernielsentikaer) [inline extensions/codeforces-extension/package.json:16]:
```suggestion
      "subtitle": "Codeforces",
```

File: extensions/codeforces-extension/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      "description": "Provides details about a handle",
+      "mode": "view"
+    },
+    {
+      "name": "search-contest",
+      "title": "Search Contest",
+      "subtitle": "Seach Upcoming and Completed Contests",
```
Comment (pernielsentikaer) [inline extensions/codeforces-extension/package.json:23]:
```suggestion
      "subtitle": "Codeforces",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Could you look into why both pushes has some initial data before showing the list? 

https://github.com/raycast/extensions/assets/38033107/f0060c7d-aecd-4d09-9cd5-43838dbfd0f2

- Could you look into the CI errors?

- Could you upload [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) to the PR

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/codeforces-extension/package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      "subtitle": "Codeforces",
+      "description": "Provides details about a handle",
+      "mode": "view"
+    },
+    {
+      "name": "search-contest",
+      "title": "Search Contest",
+      "subtitle": "Codeforces",
+      "description": "Lists upcoming and completed contests",
+      "mode": "view"
```
Comment (pernielsentikaer) [inline extensions/codeforces-extension/package.json:25]:
```suggestion
      "mode": "view",
      "arguments": [
        {
          "name": "handle",
          "placeholder": "Handle...",
          "type": "text",
          "required": true
        }
      ]
```

I would use an argument instead so you can save one view using [arguments](https://developers.raycast.com/information/lifecycle/arguments) - could you look into that?

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**